OSSM-6063 [DOC] OpenShift Service Mesh Console (OSSMC) plugin Integration (#88446)
openshift/openshift-docs: modules/ossm-uninstall-console-plugin-ocp-web-console.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin-assembly.adoc

:_mod-docs-content-type: PROCEDURE
[id="ossm-uninstall-console-plugin-ocp-web-console_{context}"]

> = Uninstalling {SMPluginShort} via {ocp-short-name} web console

Reviewer: ```suggestion
= Uninstalling {SMPluginShort} by using the web console
```

> You can uninstall the {SMPlugin} using the {ocp-short-name} web console.

Reviewer: ```suggestion
You can uninstall the {SMPlugin} by using the {ocp-product-title} web console.
```

.Procedure

. Navigate to *Installed Operators*.

. Click *Kiali Operator*.

. Select the *{OSSMC}* tab.

. Click *Delete OSSMConsole* option from the entry menu.

> . Confirm with *delete* in the modal confirmation message.

Reviewer: ```suggestion
. Confirm that you want to delete the plugin.
```
